packages/evm/test/payments/payments-distribution.js

```diff
@@ -0,0 +1,437 @@
+              const governorShare = bn(0)
+
+              itDistributesJurorFeesCorrectly(juror, governorShare, jurorShare)
+            })
+
+            context('when the governor share is greater than zero', async () => {
+              const governorShare = bn(100) // 1%
+
+              itDistributesJurorFeesCorrectly(juror, governorShare, jurorShare)
+            })
+          })
+
+          context('when the claiming juror was not active yet', async () => {
+            const juror = jurorMidPeriod1
+
+            beforeEach('execute payments', executePayments)
+
+            it('estimates juror fees correctly', async () => {
+              const fees = await paymentsBook.getJurorFees(periodId, juror, token.address)
+
+              assertBn(fees, 0, 'juror fees does not match')
+            })
+
+            it('does not transfer any fees', async () => {
+              const previousBalance = await token.balanceOf(paymentsBook.address)
+
+              await paymentsBook.claimJurorFees(periodId, token.address, { from: juror })
+
+              const currentBalance = await token.balanceOf(paymentsBook.address)
+              assertBn(currentBalance, previousBalance, 'payments book balance does not match')
+            })
+          })
+        })
+      })
+
+      context('when requesting the current period', () => {
+        const periodId = 1
+
+        beforeEach('execute payments', executePayments)
+
+        it('reverts', async () => {
+          await assertRevert(paymentsBook.claimJurorFees(periodId, token.address, { from: jurorPeriod0Term1 }), PAYMENTS_BOOK_ERRORS.NON_PAST_PERIOD)
+          await assertRevert(paymentsBook.claimJurorFees(periodId, token.address, { from: jurorPeriod0Term3 }), PAYMENTS_BOOK_ERRORS.NON_PAST_PERIOD)
+          await assertRevert(paymentsBook.claimJurorFees(periodId, token.address, { from: jurorMidPeriod1 }), PAYMENTS_BOOK_ERRORS.NON_PAST_PERIOD)
+        })
+      })
+
+      context('when requesting a future period', () => {
+        const periodId = 2
+
+        beforeEach('execute payments', executePayments)
+
+        it('reverts', async () => {
+          await assertRevert(paymentsBook.claimJurorFees(periodId, token.address, { from: jurorPeriod0Term1 }), PAYMENTS_BOOK_ERRORS.NON_PAST_PERIOD)
+          await assertRevert(paymentsBook.claimJurorFees(periodId, token.address, { from: jurorPeriod0Term3 }), PAYMENTS_BOOK_ERRORS.NON_PAST_PERIOD)
+          await assertRevert(paymentsBook.claimJurorFees(periodId, token.address, { from: jurorMidPeriod1 }), PAYMENTS_BOOK_ERRORS.NON_PAST_PERIOD)
+        })
+      })
+    })
+
+    context('when there were no payments', () => {
+      context('when requesting a past period', () => {
+        const periodId = 0
+
+        it('ignores the request', async () => {
+          const previousBalance = await token.balanceOf(paymentsBook.address)
+
+          await paymentsBook.claimJurorFees(periodId, token.address, { from: jurorPeriod0Term1 })
```

Comment:
Similarly, may be worth double checking no events are emitted.

Reply:
Addressed in b3b4648025045022dd2b426518dc3c6f756bc195